certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1301 @@
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/PaginatedDatasetList"
+            }
+          }
+        }
+      }
+    },
+    "/api/v2/documents/": {
+      "post": {
+        "description": "Uploads a file to a Data Inbox tenant for processing.",
+        "x-ms-summary": "Uploads a file to a Data Inbox tenant for processing.",
+        "summary": "Upload Document File",
+        "tags": [
+          "documents"
+        ],
+        "operationId": "documents_create",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "name": "removed",
+            "in": "formData",
+            "type": "boolean",
+            "description": "removed",
+            "x-ms-summary": "Removed Boolean"
+          },
+          {
+            "name": "dataset",
+            "in": "formData",
+            "type": "string",
+            "description": "This should usually be the dataset uuid for Inbox.",
+            "x-ms-summary": "Dataset UUID"
+          },
+          {
+            "name": "subject",
+            "in": "formData",
+            "type": "string",
+            "description": "subject",
+            "x-ms-summary": "Subject"
+          },
+          {
+            "name": "sender",
+            "in": "formData",
+            "type": "string",
+            "description": "sender",
+            "x-ms-summary": "Sender"
+          },
+          {
+            "name": "folder",
+            "in": "formData",
+            "type": "string",
+            "description": "folder",
+            "x-ms-summary": "Folder"
+          },
+          {
+            "name": "file",
+            "in": "formData",
+            "required": true,
+            "type": "file",
+            "description": "File Name",
+            "x-ms-summary": "File Name"
+          },
+          {
+            "name": "targets",
+            "in": "formData",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "targeted pattern names",
+            "x-ms-dynamic-values": {
+              "operationId": "pipelines_list",
+              "value-collection": "results",
+              "value-path": "name",
+              "value-title": "name"
+            },
+            "x-ms-dynamic-list": {
+              "operationId": "pipelines_list",
+              "itemsPath": "results",
+              "itemValuePath": "name",
+              "itemTitlePath": "name"
+            },
+            "x-ms-summary": "Targeted Patterns"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "No response body",
+            "headers": {}
```

Comment:
@tori-alkymi Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.